docs/usecases_content.rst

```diff
@@ -22,31 +24,22 @@ Namely, we will go through the following:
 +----------------------------------------------------------------------------------------------------+----------------------------------------------------------------------+
 
 
-Important reminders
-^^^^^^^^^^^^^^^^^^^
+Important remarks
+^^^^^^^^^^^^^^^^^
 
-A compulsory prerequisite for running the examples presented in the following sections is
-to have :code:`integration-service` correctly installed as explained in detail
+A compulsory prerequisite for running the examples of the following sections is
+to have *eProsima Integration-Service* correctly installed as explained in detail
 in the introductory section :ref:`Getting Started <getting started>`.
 Please make sure to follow all the steps described in the document before proceeding.
 
-Also notice that for being able to execute :code:`integration-service` with the :code:`soss` command at the end of
+Also notice that, for being able to execute *eProsima Integration-Service* with the :code:`soss` command at the end of
 each example,
 the shell must be fully overlaid with the sourcing of any colcon-built package required by the specific
```

Comment:
**any** colcon-built package required?
Maybe "**all** colcon-built packages required"?
"Any" doesn't sound good to me, but I may be wrong.

Reply:
ok. nice.